Makefile

```diff
@@ -21,36 +24,72 @@ DIR_PROBE := "$(CUR_DIR)/cmd/probe"
 build: build-runner build-probe
 
 build-runner: deps-runner
-	@cd ${DIR_RUNNER} && go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=${PROBE_VERSION}'" -o "$(CUR_DIR)/bin"
+	@cd $(DIR_RUNNER) && go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=$(PROBE_VERSION)'" -o "$(CUR_DIR)/bin"
 
 build-probe: deps-probe
-	@cd ${DIR_PROBE} && go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=${PROBE_VERSION}'" -o "$(CUR_DIR)/bin"
+	@cd $(DIR_PROBE) && go build -ldflags="-X 'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=$(PROBE_VERSION)'" -o "$(CUR_DIR)/bin"
 
+.PHONY: clean
 clean:
 	@rm -f "$(CUR_DIR)/bin/"*
 
+.PHONY: deps deps-runner deps-probe
+
+# Default kind cluster name if not overridden
+KIND_CLUSTER_NAME ?= "nethax"
+
 deps: deps-runner deps-probe
 
 deps-runner:
-	@cd ${DIR_RUNNER} && go mod download
+	@cd $(DIR_RUNNER) && go mod download
 
 deps-probe:
-	@cd ${DIR_PROBE} && go mod download
+	@cd $(DIR_PROBE) && go mod download
 
 
+.PHONY: docker docker-runner docker-probe
 docker: docker-runner docker-probe
 
 docker-runner:
-	@docker build -f Dockerfile-runner --build-arg PROBE_VERSION=${PROBE_VERSION} -t nethax-runner:${RUNNER_VERSION} .
+	@docker build -f Dockerfile-runner --build-arg PROBE_VERSION="'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=$(PROBE_VERSION)'" -t nethax-runner:$(RUNNER_VERSION) .
 ifndef CI
-	@kind load docker-image nethax-runner:${RUNNER_VERSION} || true
+	@kind load docker-image --name $(KIND_CLUSTER_NAME) nethax-runner:$(RUNNER_VERSION) || true
 endif
 
 docker-probe:
-	@docker build -f Dockerfile-probe --build-arg PROBE_VERSION=${PROBE_VERSION} -t nethax-probe:${PROBE_VERSION} .
+	@docker build -f Dockerfile-probe --build-arg PROBE_VERSION="'github.com/grafana/nethax/pkg/kubernetes.ProbeImageVersion=$(PROBE_VERSION)'" -t nethax-probe:$(PROBE_VERSION) .
 ifndef CI
-	@kind load docker-image nethax-probe:${PROBE_VERSION} || true
+	@kind load docker-image --name $(KIND_CLUSTER_NAME) nethax-probe:$(PROBE_VERSION) || true
 endif
 
+.PHONY: test
 test:
 	@go test ./...
+
+.PHONY: kind-init-oteldemo
+
+# Re-initializes a kind cluster with the OTel demo
+kind-init-oteldemo:
+	@kind delete cluster --name $(KIND_CLUSTER_NAME) || true
+	@kind create cluster --name $(KIND_CLUSTER_NAME)
+	kubectl --context "kind-$(KIND_CLUSTER_NAME)" create ns otel-demo
+	kubectl --context "kind-$(KIND_CLUSTER_NAME)" apply -n otel-demo -f https://raw.githubusercontent.com/open-telemetry/opentelemetry-demo/main/kubernetes/opentelemetry-demo.yaml || true
+	kubectl --context "kind-$(KIND_CLUSTER_NAME)" create cm -n otel-demo grafana-dashboards
+	kubectl --context "kind-$(KIND_CLUSTER_NAME)" replace -n otel-demo -f https://raw.githubusercontent.com/open-telemetry/opentelemetry-demo/main/kubernetes/opentelemetry-demo.yaml
```

Comment:
Any reason why this commands aren't preceded with `@`? Or why not even removing the `@` from the commands above and have `make kind-init-oteldemo` spit out all the executed commands?

Reply:
I can do that, I just like having them printed so I can see what's going on, but I'll change it 🤷🏼‍♀️

Reply:
FWIW I also like them being printed 😸 I was just asking because it didn't feel consistent with the previous commands being hidden.